apps/condo/domains/meter/schema/Meter.test.js

```diff
@@ -813,16 +812,13 @@ describe('Meter', () => {
             // NOTE: give worker some time
             await sleep(1000)
 
-            const testFunc = async () => {
-                await Meter.getAll(client, { id: meter.id })
-            }
-
             // Test access after residents disconnected in worker
             // NOTE: This place is unstable. Because of parallel sub-requests inside Meter.getAll
             // we get different path in error payload, and get either organization or property error
             // in random order from request to request. So we skip testing path/value here.
             // Exception message and error type is enough.
-            await expectToThrowAccessDeniedError(testFunc, null)
+            const metersAfterResidentReconnects = await Meter.getAll(client, { id: meter.id })
```

Comment:
why not use `find` here too?

please add empty row after `const`blocks

Reply:
in tests we usually use api from `testSchema` of this domain